app/main/views/index.py

```diff
@@ -16,11 +17,14 @@ def landing(service_id, document_id):
     except HTTPError as e:
         abort(e.status_code)
 
+    service_contact_link = service['data']['contact_link']
```

Comment:
It might be worth calling these variables something like `service_contact_details` and `contact_details_type` since they could be email addresses or phone numbers.

(There's a bit of the story to change the names of things in API too, which might help make things clearer since this would remove all mentions of links and let the right hand side become something like `service['data']['contact_details']`)

Reply:
I was thinking contact_link doesn't sound most true to the meaning, but wanted to stay consistent. It's good to know we want to change that field name. I will change those variable names as you suggested ☺️